pkg/security/metrics/prometheus/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package prometheus contains code for injecting namespace allocation related metrics
```

Comment:
Nit: just `pkg/security/metrics` for this package name like we do that elsewhere.

Reply:
done